src/xbot/common/math/XYPair.java

```diff
@@ -79,6 +79,10 @@ public XYPair add(XYPair pair) {
         this.y += pair.y;
         return this;
     }
+    
+    public double getDistanceToPoint(XYPair otherPoint) {
+        return Math.sqrt(Math.pow((otherPoint.x - this.x), 2) + Math.pow((otherPoint.y - this.y), 2));
```

Comment:
This could be made more concise by either subtracting the vectors and getting the magnitude or using `Math.hypot`